google-cloud-storage/Rakefile

```diff
@@ -176,6 +176,40 @@ namespace :ci do
   end
 end
 
+task :samples do
+  Rake::Task["samples:latest"].invoke
+end
+
+namespace :samples do
+  task :latest do
+    if File.directory? "samples"
```

Comment:
Why check for the `samples` dir when we know it's there? (Same in `master` task.)

Reply:
copied from the generated rakefile, removed